### PR TITLE
Rails migrations

### PR DESCRIPTION
**Pull request by iamvery**

### CODE REVIEW

File: _drafts/rails-migrations.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+We had a lively conversation today at [Big Nerd Ranch][bnr] about Rails migrations.
+What are they really for?
+How long should you keep them around?
+What about this crazy schema.rb file?!
+TIL WE HAZ OPINIONS!
+
+⚠️ The views of this post are my own.
+This is in no away a representation of our team's view of the subject.
+Quite the contrary, actually.
```
Comment (CodingItWrong) [inline _drafts/rails-migrations.md]: 😂

File: _drafts/rails-migrations.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+* your coworker's ⤴️
+* staging env(s)
+* production
+
+With that, you are able to avoid a lot of the pain of managing database structure across your team and deployment.
+
+### Can I delete them?
+
+It depends (doesn't it always).
+Test Double says, ["Keep them working"][td].
```
Comment (CodingItWrong) [inline _drafts/rails-migrations.md]: It's a bit confusing that you say "it depends" here and then only give one view at first. The other view isn't given until line 57.

I would lean more towards ordering the post like this:

1. Introduce the question (can I delete migration files?)
2. Define concepts (migration files, schema file)
3. Give different views (Rails and Test Double)
4. Work out the implications of the different views, and how they lead you to a different answer on the question

Whereas you introduce one view, dive into defining the concepts, then go to the other view. If your order is intentional, would love to hear your thoughts on it!

File: _drafts/rails-migrations.md (hunk 1 above)
Comment (iamvery) [inline _drafts/rails-migrations.md]: You can sort of see by that that I expected to come to the conclusion "delete migrations", but that didn't happen. So I'd say we all agree more than not to the actual conclusion of the post. Maybe I'll soften that a bit?

File: _drafts/rails-migrations.md (hunk 2 above)
Comment (iamvery) [inline _drafts/rails-migrations.md]: Super legit. Thanks for the feedback! The ordering is bugging me too. :heart:

File: _drafts/rails-migrations.md (hunk 1 above)
Comment (CodingItWrong) [inline _drafts/rails-migrations.md]: I was thinking that, but forgot to say something about it. Yeah, I don't think there was any disagreement about the implications of the two different views. You can definitely say which side you lean more towards personally (unless you're now more confused than when we started!)

Comment (iamvery) [inline _drafts/rails-migrations.md]: ┻━┻︵ \(°□°)/ ︵ ┻━┻

**Pull request closed without merging**